source/touchTracker.py

```diff
@@ -313,7 +341,7 @@ def emitTrackers(self):
 			#Only emit trackers if there are not unknown actions
 			hasUnknownTrackers=self.numUnknownTrackers
 			if not hasUnknownTrackers:
-				for tracker in list(self.multiTouchTrackers):
+				for tracker in self.multiTouchTrackers:

```

Comment:
The list does still need to be copied here when looping through it as the original list may be mutated in the body of the loop. 
If you don't like `list(x)` then perhaps `x.copy()` ?

Reply:
this change was a mistake when attempting to update types. I do think that would be more explicit and avoid future issues like this.

```suggestion
				for tracker in self.multiTouchTrackers.copy():
```